notebook_data_redirector/common.py

```diff
@@ -141,12 +179,13 @@ def iterate_files(folder):
                 # Here we're recursively calling iterate_files on a nested folder and
                 # receiving an iterator that contains all of its files.  "yield from"
                 # will yield each value from that iterator in turn.
-                yield from iterate_files(item)
+                yield from iterate_files(item, shared=shared or is_box_file_public(item.get()))
             elif item.object_type == "file":
-                yield item
+                yield item, shared
         if count >= GET_ITEMS_LIMIT:
             offset += count
         else:
+            # this clause not covered by testing, but I think it's because folder.get_items works properly
```

Comment:
That's puzzling, isn't this break statement the only way to exit the while loop?

Reply:
oh good point.... I don't know why pytest's coverage test reports this line as missing.